pulp_puppet_plugins/pulp_puppet/plugins/distributors/installdistributor.py

```diff
@@ -147,11 +180,7 @@ def publish_repo(self, repo, publish_conduit, config):
                 _('failed to move temporary destination to destination directory: %s') % str(e),
                 self.detail_report.report)
 
-        # return some kind of report
-        if self.detail_report.has_errors:
-            return publish_conduit.build_failure_report(_('failed'), self.detail_report.report)
```

Comment:
because if line 151 was True, line 131 would also have been True, and would have returned.